src/formatters/teamcity.ts

```diff
@@ -0,0 +1,53 @@
+import { Dictionary } from '@stoplight/types';
+import { IRuleResult } from '../types';
+import { Formatter } from './types';
+import { getSeverityName, groupBySource, sortResults } from './utils';
+
+function escapeString(str?: string | number) {
+  if (!str) {
```

Comment:
What about `0`?
Let's adjust the condition a bit, suggest the following
```suggestion
  if (str === void 0) {
    return '';
  }
```